firstDirectory/topic6/Main.java

```diff
@@ -0,0 +1,12 @@
+package firstDirectory.topic6;
+
+public class Main {
+    public static void main(String[] args) {
+        ContulPersoanei cont = new ContulPersoanei("John", 100, 500);
+        //cont.initialBalanceSet(300);
```

Comment:
sugerez sa adaugi ceva opertiuni care arunca exceptii intr-un try si sa le correctezi in catch